sequences/convert.go

```diff
@@ -30,6 +42,8 @@ func convert(inpath, outpath string, scripts ParsedScripts, locals Locals) error
 	}
 	locals.Global["totalLayers"] = float64(preflightResults.totalLayers)
 	locals.Global["totalTime"] = float64(preflightResults.totalTime)
+	coolingModuleSpeedPercentage := locals.PerExtruder["coolingModuleSpeedPercentage"]
+	enableCoolingModuleAtLayer := locals.PerExtruder["enableCoolingModuleAtLayer"]
```

Comment:
Since `locals.PerExtruder` are all floats, and these variables aren't intended to be available in scripts but doing it this way makes them available, I think we should simplify things a bit and just add these two settings to the `parsedScripts` struct as

```go
type ParsedScripts struct {
	Start                        printerscript.Tree
	End                          printerscript.Tree
	LayerChange                  printerscript.Tree
	MaterialChange               []printerscript.Tree
	CoolingModuleSpeedPercentage []int
	EnableCoolingModuleAtLayer   []int
}
```

This would
- avoid doing `int()` casts everwhere
- avoid floating-point precision errors
- avoid exposing these undocumented variables in user scripts